crates/bevy_render/src/lib.rs

```diff
@@ -58,7 +58,13 @@ use renderer::{AssetRenderResourceBindings, RenderResourceBindings, RenderResour
 use shader::ShaderLoader;
 #[cfg(feature = "hdr")]
 use texture::HdrTextureLoader;
-#[cfg(feature = "png")]
+#[cfg(any(
```

Comment:
You should do something similar for the `AudioPlugin` around the `init_asset_loader::<Mp3Loader>()`